### DISCUSSION TITLE
updating to windows 11 build 26080.1  and 26085.1 results in a blank screen after restart

### DISCUSSION THREAD

**Discussion by prosperot** [utmapp/UTM#6232] [Windows]
**Describe the issue**  
updating to windows 11 ARM build 26080.1  or 26085.1 results in a black screen after restart. no error / crash messages, just the black (black) screen

on previous W11 builds everything worked perfectly

**Configuration**  
* UTM Version: 4.4.4 (93)
* macOS Version: 14.3.1
* Mac Chip (Intel, M1, ...): M2

**Comment by osy** [comment]
I was able to update to 26090.1 perfectly fine. After it restarts, I do see a black screen but after waiting ~5 minutes, it restarted again and was fine.
